operations/sftpgo-authentication-service/internal/service/subscription_test.go

```
@@ -0,0 +1,169 @@
+
+func TestSubscriptionService_GetUserFolderList(t *testing.T) {
+	logger := log.NewAppLogger("INFO")
+
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.URL.Query().Get("customerEmail") {
+		case "valid@example.com":
+			resp := models.FolderResponse{
+				IsValidCustomer: true,
+				ProjectKeys:     []string{"PROJ1", "proj2"},
+			}
+			json.NewEncoder(w).Encode(resp)
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)